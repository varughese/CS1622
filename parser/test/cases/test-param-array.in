void foo(int arr[])
{
}

void main(void) {}
