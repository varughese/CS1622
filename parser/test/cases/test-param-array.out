[program
  [fun-declaration
    [void]
    [foo]
    [params 
      [param [int] [arr] [\[\]]]]
    [compound-stmt 
    ]
  ]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
    ]
  ]
]
